library/model/src/main/java/org/cru/godtools/model/TranslationFile.kt

```diff
@@ -2,9 +2,9 @@ package org.cru.godtools.model
 
 import org.cru.godtools.model.Base.INVALID_ID
 
-class TranslationFile constructor(
+data class TranslationFile(
```

Comment:
What does ```data``` add to this?

Reply:
data classes automatically generate an `equals`, `hashCode`, and `toString` methods for this class